Module3/Models/Product.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DeliveryService.Models
+{
+    public class Product : BaseModel
+    {
+        public string Name { get; set; }
```

Comment:
seems like it should be OrderId

Reply:
OrderId instead of Name? Why? I dont think i even need OrderId property here.

Reply:
not instead 
to add
it will need for EF relation

Reply:
up to u